lib/fizzy/stack.hpp

```diff
@@ -58,32 +58,54 @@ class Stack : public std::vector<T>
 
 class OperandStack
 {
+    /// The size of the pre-allocated internal storage: 128 bytes.
+    static constexpr auto small_storage_size = 128 / sizeof(uint64_t);
+
     /// The pointer to the top item, or below the stack bottom if stack is empty.
     ///
     /// This pointer always alias m_storage, but it is kept as the first field
     /// because it is accessed the most. Therefore, it must be initialized
     /// in the constructor after the m_storage.
     uint64_t* m_top;
 
-    /// The storage for items.
-    std::unique_ptr<uint64_t[]> m_storage;
+    /// The bottom of the stack. Set in the constructor and never modified.
+    ///
+    /// TODO: This pointer is rarely used and may be removed.
```

Comment:
Is this comment still valid?

Reply:
Yes, I just added it. There is some experimentation needed for it, with probably no performance implications. But it is worth just for sanity and "simpler" design.